src/index.js

```diff
@@ -490,6 +505,7 @@ export default class Carousel extends Component {
               { this.renderSlides() }
             </ul>
           </div>
+          <div className='control-div' style={{ ...arrowStyle }}>
```

Comment:
I don't think we should be applying arrow-specific styling to the div that wraps all controls - this would affect any custom controls passed in by the parent component as well. Would be better to only apply these additional styles to the arrow(s) themselves if possible.

Reply:
good point, I realized just now that the dots was broken because of this div. Have pushed a fix.

Reply:
Have removed this, instead passing the styles to the custom arrow component.